docs/planning/sprintplanning.md

```diff
@@ -0,0 +1,66 @@
+# Knative Eventing sprint planning
+
+This document summarizes project planning mechanics for Knative Eventing releases. Knative Eventing release is planned every 6 weeks and will continue to be so. Current proposal is to align sprint with the release cycle of 6 weeks. Eventually this may change as we learn from the process.
+
+## How to manage Bugs?
+- Submit new bugs by creating a new issue of type “Report Bug”.
```

Comment:
Report Bug is the standard template name in GitHub - I concluded so because when I created a new Issue template in Github it gave this name by default, I could be wrong here.. I'll rename it to Bug when creating the template if Prow uses it.

Reply:
Oh I understand now, you're talking about the issue creation template rather than the label. Disregard my comment!